internal/httpserver/httpserver.go

```diff
@@ -36,6 +39,10 @@ type Options struct {
 	ReadTimeout         time.Duration // HTTP Server read timeout.
 	WriteTimeout        time.Duration // HTTP Server write timeout.
 	ConfigPath          string        // Config path.
+	DelayParticipation  float32       // Delay participation rate (percentage of requests that will be delayed. 0.0 <= p <= 1.0).
```

Comment:
```suggestion
	DelayParticipation  float32       // Delay participation rate (fraction of requests that will be delayed. 0.0 <= p <= 1.0).
```
(also below)

This is a `float32` because of the backing library?

Reply:
Yes.

https://pkg.go.dev/github.com/lingrino/go-fault#WithParticipation